department-of-veterans-affairs/vets-api
# Toxic Exposure: Refactor TE flag to indicate PDF version

This PR renames the `includeToxicExposure` flag to `startedFormVersion`, and refactors the logic to indicate which PDF version the user started the 526 application with (2019 or 2022).


## Summary

- *This work is behind a feature toggle (flipper): YES*
- add feature flag for Toxic Exposure 1.1: `disability_526_toxic_exposure_ipf`
- adds `startedFormVersion` logic to 1.0 (new form) and 1.1 (IPF) workflows

## Related issue(s)

- [#85625](https://app.zenhub.com/workspaces/disability-experience-63dbdb0a401c4400119d3a44/issues/gh/department-of-veterans-affairs/va.gov-team/85625)

## Testing done

- Toxic Exposure 1.0 should work as normal with a new form created (`formData.startedFormVersion: "2022"`)
- how to test Toxic Exposure 1.1 in the frontend:
  - turn off all TE flags
  - create a new form
  - ensure `formData `does not have a `startedFormVersion `attribute
  - with the user having an IPF, turn on the `disability_526_toxic_exposure_ipf` flag
  - return to the IPF
  - ensure `formData` does have a `startedFormVersion` attribute as 2019

## Screenshots
![image](https://github.com/department-of-veterans-affairs/vets-api/assets/19178435/5eb4a5a6-b67d-4a18-9fd8-4047f644aa0e)

## What areas of the site does it impact?
- Form526 toxic exposure indicator

## Acceptance criteria
